### PR TITLE
US34 Task #101

### PR DESCRIPTION
**Pull request by jaymerut**

### CODE REVIEW

File: temp/App.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
+package main.java.memoranda.ui;
```
Comment (souch3) [inline temp/App.java]: Jayme, do you know what is causing this file to generate? It isn't in the repo at all right now and I don't know that it should be tracked in the repo

Comment (jaymerut) [reply temp/App.java]: You are referring to the temp.App.java file correct?

Comment (souch3) [reply temp/App.java]: Yes, the fact that it doesn't currently exist and showed up in a temp folder makes me think it isn't a file we should be tracking in the repo.

Comment (jaymerut) [reply temp/App.java]: I'm not sure how it got created. I wonder if it's possible that the file gets created automatically if a project gets created and saved and something gets stored as a temp file? I don't think it's a necessary file in that case.

Comment (souch3) [reply temp/App.java]: Yeah, I was thinking it came from running the build. I'd say try removing it and running the app, if it works, make a new commit and push it up to here. Then it should be good to go

**Comment by jaymerut** [comment]
Sounds good I'll do that

**Pull request merged**